river/utils/math.py

```diff
@@ -353,3 +354,27 @@ def woodbury_matrix(A: np.ndarray, U: np.ndarray, V: np.ndarray):
     eye = np.eye(len(V))
     Au = A @ U
     A -= Au @ np.linalg.inv(eye + V @ Au) @ V @ A
+
+
+def log_sum_2_exp(a: float, b: float) -> float:
+    """Computation of log( (e^a + e^b) / 2) in an overflow-proof way
+
+    Parameters
+    ----------
+    a : float
```

Comment:
There is no need to specify the parameter type here :D

Reply:
Solved in latest commit.